packages/app/features/Send/InformationInput/components/TransactionFee/AbstractedTransactionFee.tsx

```diff
@@ -0,0 +1,243 @@
+import type { FC } from 'react';
+import { useEffect, useRef, useState } from 'react';
+import {
+	ActivityIndicator,
+	Image,
+	StyleSheet,
+	TouchableOpacity,
+} from 'react-native';
+import type { Token, TransactionPayload } from '@walless/core';
+import { Networks } from '@walless/core';
+import { BindDirections, modalActions, Text, View } from '@walless/gui';
+import { ChevronDown, Exclamation } from '@walless/icons';
+import type { CollectibleDocument, TokenDocument } from '@walless/store';
+import { useSnapshot } from 'valtio';
+
+import {
+	injectedElements,
+	transactionActions,
+	transactionContext,
+} from '../../../../../state/transaction';
+
+import TokenFeeDropDown from './TokenFeeDropDown';
+
+interface Props {
+	tokenForFee?: Token;
+	payload?: TransactionPayload;
+	feeText?: string;
+}
+
+export const AbstractedTransactionFee: FC<Props> = () => {
+	const [isDropped, setIsDropped] = useState(false);
+	const [isFeeLoading, setIsFeeLoading] = useState(false);
+	const [error, setError] = useState<string>('');
```

Comment:
I guess `<string>` is not necessary if `error` already initialized with empty string, check this again